css/styles.css

```
@@ -34,4 +34,623 @@ body {
+  color: #000000;
+}
+
+a {
+  text-decoration: none;
+}
+
+.btn {
+  font-family: "Raleway", "Arial", sans-serif;
+  font-weight: 800;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 21px;
+  letter-spacing: 0.2em;
+  text-align: center;
+  vertical-align: middle;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000000;
+  border: none;
+
+  /*background-image: url(/img/decor/services-tabs-buttons_rest.svg);*/
+  background-repeat: no-repeat;
+  background-position: 50%;
+  background-color: inherit;
+
+  width: 160px;
+}
+
+.visually-hidden {
+  position: absolute;
+  clip: rect(0 0 0 0);
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  overflow: hidden;
+}
+
+.link-for-page {
+  font-family: "Raleway", "Arial", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 18px;
+  line-height: 21px;
+  text-align: center;
+  letter-spacing: 0.2em;
+  text-transform: uppercase;
+  color: #000000;
+  /*background-image: url(/img/decor/link-for-page.svg);*/
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+/* Главная навигация */
+/*Основное меню*/
+.navigation-link {
+  font-family: "Raleway", "Arial", sans-serif;
+  font-weight: 800;
+  font-style: normal;
+  text-align: center;
+  line-height: 21px;
+  letter-spacing: 0.2em;
+  text-transform: uppercase;
+
+  color: #000000;
+}
+/*Пользовательское меню*/
+.navigation-link-user {
+  font-family: "Rubik", "Arial", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 19px;
+
+  color: #000000;
+}
+
+.navigation-link-logout {
+  font-family: "Rubik", "Arial", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 19px;
+
+  color: #000000;
+  opacity: 0.3;
+}
+
+/*Основное меню_саб-меню (скрытое)*/
+.sub-menu-cat a {
+  font-family: "Rubik", "Arial", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #000000;
+}
+
+/*
+ ##################
+  Главная страница
+ ##################
+*/
+
+/*Слайдер*/
+
+.slider-part {
+  font-family: "Rubik", "Arial", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 180px;
+  line-height: 30px;
+
+  width: 250px; /*Временно*/
+  height: 135px; /*Временно*/
+
+  background-color: #ffe17f; /*Временно*/
+  color: #ffffff;
+
+  display: flex; /*Временно*/
+  align-items: center; /*Временно*/
+  text-align: right; /*Временно*/
+}
+
+.slider-link {
+  display: block; /*Временно*/
+  width: 220px;
+  height: 20px;
+}
+
+.slider-table-specifications th {
+  font-family: "Rubik", "Arial", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 36px;
+  line-height: 30px;
+
+  color: #000000;
+}
+
+.slider-table-specifications td {
+  font-size: 16px;
+}
+
+/* Кнопки переключения галереи слайдера*/
+.slider-button {
+  width: 20px;
+  height: 60px;
+}
+
+.slider-prev {
+  background-image: url(/img/decor/slider-prev.svg);
```

```suggestion
  background-image: url(../img/decor/slider-prev.svg);
```
так как путь относительный.